tests/unit/test_estimator.py

```diff
@@ -325,15 +325,27 @@ def test_init_with_source_dir_s3(strftime, sagemaker_session):
 
 
 def test_framework_name_from_framework_image():
-    framework, py_ver = framework_name_from_image('123.dkr.ecr.us-west-2.amazonaws.com/sagemaker-mxnet-py2-gpu:1')
+    framework, py_ver, image_ver = \
+        framework_name_from_image('123.dkr.ecr.us-west-2.amazonaws.com/sagemaker-mxnet-py2-gpu:1.0')
     assert framework == 'mxnet'
     assert py_ver == 'py2'
+    assert image_ver == '1.0'
 
 
 def test_framework_name_from_other():
-    framework, py_ver = framework_name_from_image('123.dkr.ecr.us-west-2.amazonaws.com/sagemaker-myown-py2-gpu:1')
+    framework, py_ver, image_ver = \
+        framework_name_from_image('123.dkr.ecr.us-west-2.amazonaws.com/sagemaker-myown-py2-gpu:1.0')
     assert framework is None
     assert py_ver is None
+    assert image_ver is None
+
+
+def test_framework_name_from_image_wrong_version():
```

Comment:
Are these tests suppose to test parsing of the image name? If so, what is wrong with 1.5 version? I think 'framework_name_from_image' shall just return 1.5.